clarifai/models/model_serving/README.md

```diff
@@ -14,19 +14,26 @@ $ clarifai-model-upload-init --model_name <Your model name> \
 ```
 2. Edit the `requirements.txt` file with dependencies needed to run inference on your model and the `labels.txt` (if available in dir) with the labels your model is to predict.
 3. Add your model loading and inference code inside `inference.py` script of the generated model repository under the `setup()` and `predict()` functions respectively. Refer to  The [Inference Script section]() for a description of this file.
-4. Generate a zip of your triton model for deployment via commandline.
+4. Testing your implemtation locally by running `<your_triton_folder>/1/test.py` with basic predefined tests.
+To avoid missing dependencies when deploying, recommend to use conda to create clean environment. Then install everything in `requirements.txt`. Follow instruction inside [test.py](./models/test.py) for implementing custom tests. Running the test by using pytest:
```

Comment:
*"Run the test by using pytest:"

Reply:
Updated readme with building env instruction